### PR TITLE
infectious agent survival in environment

### PR DESCRIPTION
**Pull request by miseminger**
imported names of all 7 coronaviruses known to infect humans as well as two more unidentified human coronaviruses from CIDO as subclasses of 'Viruses' (no generic 'human coronavirus' term in OLS)

imported 'environmental material' and children:
-droplet
-aerosol
-dust
-haven't added 'surface' yet, as most surfaces may fall under 'fomite'

added invented term as a subclass of 'survival time':
-'infectious agent survival time in environment'

### CODE REVIEW

**Comment by miseminger** [comment]
Looking at this a little more, I'm not too clear on how to decide which questions fall under "infectious agent transmission process affected by _______" and which fall under "infectious agent survival time in ________".  The decision tree questions include RQs like "What is the survival time of SARS-COV-2 as an airborne aerosol?" (falls under survival time) but also some like "Spreading SARS-COV-2 through ocular fluids" which is (implicitly) about survival time in ocular fluids as well as (explicity) about transmission by ocular fluids.  Shall I sort questions into either bin according to the most explicitly-stated topic, or find a way to tie them together?

File: src/ontology/decision-edit.owl [hunk 1]
```diff
@@ -3912,6 +4011,82 @@ BUT: diseases have phases - through latent/incubation/contagious/ initial / subs
     
 
 
+    <!-- http://purl.obolibrary.org/obo/RO_067d925e_a7a1_475f_b6a6_adfe20d159810002092 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/RO_067d925e_a7a1_475f_b6a6_adfe20d159810002092">
```
Comment (ddooley) [inline src/ontology/decision-edit.owl]: This URI needs to be fixed - that Protege setup one must do to get the editor to issue a new ID.  If you need tips on how to get a good ID after the fact, let me know and we can do it together.

File: src/ontology/decision-edit.owl [hunk 2]
```diff
@@ -3912,6 +4011,82 @@ BUT: diseases have phases - through latent/incubation/contagious/ initial / subs
...
+ (by occupation or pre-existing health conditions)&quot;</obo:IAO_0000112>
+        <rdfs:comment xml:lang="en">change &apos;medical specialty role&apos; to more generic &apos;professional role&apos; once &apos;professional role&apos; is imported</rdfs:comment>
+        <rdfs:label xml:lang="en">transmission process by population and PPE usage</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/RO_af52161a_92a4_4f37_9860_b6c6ecb1271d0002092 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/RO_af52161a_92a4_4f37_9860_b6c6ecb1271d0002092">
```
Comment (ddooley) [inline src/ontology/decision-edit.owl]: This one too!

**Review by ddooley — COMMENTED** [review]
Aside from those two ids that need adjusting, everything else looks good.  
One other note is that the catalog.xml file doesn't need to be included in pull request.  You can block catalog.xml from the output in github desktop by right clicking on it and selecting "ignore file".

**Pull request merged**